src/lib.rs

```diff
@@ -182,7 +182,7 @@ pub fn init_and_run(options: Options) -> ! {
 
         winit::window::WindowBuilder::new()
             .with_title(BASE_WINDOW_TITLE)
-            .with_inner_size(winit::dpi::LogicalSize::new(1280.0, 720.0))
+            .with_maximized(true)
```

Comment:
Start with a maximized window

Reply:
We'll need to test this on macOS before merging. If this works well, I think we can get rid of the fullscreen (`HS_FULLSCREEN=1`) option completely.

Reply:
I'd like to keep the fullscreen option too. May be useful for some showcases.

Reply:
Just tested on linux (X11) :heavy_check_mark: